src/server/config/Passport.ts

```diff
@@ -69,7 +69,7 @@ export class PassportStrategy {
   public getJWTAdminLogin() {
     return new JwtStrategy(jwtOptions, (payload, done) => {
       this.AdminModel.findById(payload._id, (err, user) => {
-        if (err || user.deleted) {
+        if (err || !user || user.deleted) {

```

Comment:
Has this happened before? I don't understand why Passport wouldn't be giving an error nor a user.

Reply:
Not sure why I changed this. Maybe I was getting some type error? Just checked and it seems to work fine w/o. I'll switch it back.